syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
Comparing my `$profile` before and after your changes, they look different. With your changes, operators like `-not` and cmdlets like `test-path` (all of them, really) are no longer highlighted.